templates/project.html

```diff
@@ -87,7 +87,11 @@ <h2>Activate App</h2>
               <header class="usa-card__header">
                 <h2 class="usa-card__heading">
                   {% if not q.can_start_new_task and q.question.spec.app_overrides_name_and_icon is None %}
-                    {{q.task.title}}
+                    {% comment %} display the project title on this specific card {% endcomment %}
+                    {% if q.question.key == "system_basic_info" %}
+                      {{ project.title }}
+                    {% endif %}
+                    {{ q.task.title }}
```

Comment:
Let's include `System Basic Info` after `{{ project.title }}` and put the `{{ q.task.title }}` in an `{% else %}`

So
```
{% if q.question.key == "system_basic_info" %}
  {{ project.title }} System Basic Info
{% else %}
  {{ q.task.title }}
{% endif %}
```
When I run this as is with a project named *My New System* the card title appears as *My New System CMS AWS System Basic Info*

Reply:
Yup, I can add the else to make it more readable. We might want to use {{ q.task.title }} in place of the words System Basic Info though, so that we continue to pull that text content out of the database. I'll make the change and you can let me know what you think.

Reply:
Oh, and this part:

> When I run this as is with a project named _My New System_ the card title appears as _My New System CMS AWS System Basic Info_

happens because in the database, the q.task.title referenced is `CMS AWS System Basic Info`. I intended the database text would get updated to just `System Profile`. Should we handle this update differently?